be/src/io/cache/block_file_cache.cpp

```diff
@@ -796,6 +818,70 @@ bool BlockFileCache::try_reserve_for_ttl(size_t size, std::lock_guard<std::mutex
     return true;
 }
 
+bool BlockFileCache::try_reserve_for_ttl(size_t size, std::lock_guard<std::mutex>& cache_lock) {
+    if (try_reserve_for_ttl_without_lru(size, cache_lock)) {
```

Comment:
```
if (!try_reserve_for_ttl_without_lru(size, cache_lock) && config::enable_ttl_cache_evict_using_lru) {
}
```

Reply:
cannot see the improvement of code correctness and readability of such change